bindings/nodejs/tests/nodejs_int/nodejs_int.cpp

```diff
@@ -303,16 +303,16 @@ static void publish_mock_message(const v8::FunctionCallbackInfo<v8::Value>& info
 }
 
 static const char * NOOP_JS_MODULE = "module.exports = { "   \
-"create: function(broker, configuration) { "       \
-"  console.log('create'); "                            \
-"  return true; "                                      \
-"}, "                                                  \
-"receive: function(msg) { "                            \
-"  console.log('receive'); "                           \
-"}, "                                                  \
-"destroy: function() { "                               \
-"  console.log('destroy'); "                           \
-"} "                                                   \
+"create: function(broker, configuration) { "                 \
+"  console.log('create'); "                                  \
+"  return true; "                                            \
+"}, "                                                        \
+"receive: function(msg) { "                                  \
+"  console.log('receive'); "                                 \
+"}, "                                                        \
+"destroy: function() { "                                     \
+"  console.log('destroy'); "                                 \
+"} "                                                         \
```

Comment:
This is purely cosmetic. I noticed that slashes seemed to be in a column, so I just put them all in alignment.